Android/app/src/main/java/com/team1/issuetracker/ui/main/issue/adapter/IssueListAdapter.kt

```diff
@@ -0,0 +1,184 @@
+
+            binding.checkbox.setOnClickListener {
+                if(binding.checkbox.isChecked){
+                    PrintLog.printLog("checked")
+                    getItem(adapterPosition).isChecked = true
+                    binding.swipeView.setBackgroundColor(
+                        ContextCompat.getColor(
+                            binding.root.context,
+                            R.color.Backgrounds2
+                        )
+                    )
+                }
+                else{
+                    PrintLog.printLog("unchecked")
+                    getItem(adapterPosition).isChecked = false
+                    binding.swipeView.setBackgroundColor(
+                        ContextCompat.getColor(
+                            binding.root.context,
+                            R.color.white
+                        )
+                    )
+                }
+                itemCheck.invoke(getItem(adapterPosition).issueId) // IssueFragment에서 뷰모델과 연결된다, 뷰모델에는 open 상태 아닌 아이템도 다 관리하므로 issueId 넘기기!
+            }
+            // 직접 체크 박스 클릭하는 경우만 고려하기 위해 setOnCheckedChangeListener 대신 onClickListene 사용
+
+
+            binding.root.setOnLongClickListener(View.OnLongClickListener {
+                val pos = adapterPosition
+                Log.d("AppTest", "long click pos : $pos")
+
+                longClick.invoke()
+
+                return@OnLongClickListener true
+            })
+        }
+
+        fun setClamped(isClamped: Boolean) {
+            PrintLog.printLog("issue adapter/ setClamped, ${isClamped}, ${adapterPosition}")
+            getItem(adapterPosition).isSwiped = isClamped
+        }
+
+        fun getClamped(): Boolean {
+            return getItem(adapterPosition).isSwiped
+        }
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): IssueViewHolder {
+        val binding = ItemIssueBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return IssueViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: IssueViewHolder, position: Int) {
+        holder.bind(getItem(position), longClick, itemCheck, closeSwiped)
+    }
+
+    fun removeItem(position: Int) {  // currentList에서 바로 아이템 지우면 에러 발생
+        val newList = currentList.toMutableList()
+        newList.removeAt(position)
+
+        newList.forEach {
+            it.isSwiped = false
+        } // 한 아이템 삭제 시 다른 아이템들 모두 스와이프x 상태 처리하기 위함
+
+        submitList(newList.toList())
+    }
+
+    fun makeCheckBosVisible() {
+        val newList = ArrayList<Issue>()
+        currentList.forEach {
+            newList.add(
+                Issue(
+                    it.issueId,
+                    it.mileStone,
+                    it.title,
+                    it.content,
+                    it.labelContent,
+                    it.labelColor,
+                    false,
+                    true
+                )
+            )
+        }
+        submitList(newList.toList())
```

Comment:
List 자체 복사 비용은 엄청 빨리 지속적으로 일어나지만 않으면 그렇게 크지 않습니다.
요런 코드가 훨씬 좋을 수 있어요! 버그 가능성도 적구요. 
```suggestion

        submitList(currentList.map { Issue(.....) })
```